### DISCUSSION TITLE
Add _Sliding Window Mode_ (Similar to Andy's Working Notes)

### DISCUSSION THREAD

**Discussion by RoyiAvital** [bangle-io/bangle-io#219] [Ideas]
When one click on a Wiki Link `[[NoteID]]` the note will open to the right in a sliding window mode.

See [Andy Matuschak's Notes](https://notes.andymatuschak.org/About_these_notes).

**Comment by kepta** [comment]
With shift click you can open it on the right sidebar. Is that what you are looking for?

**Comment by RoyiAvital** [comment]
I am not sure.
Have you looked at the site I linked to?

**Comment by kepta** [comment]
https://user-images.githubusercontent.com/6966254/141421512-7048658b-079b-40e6-bb39-86422c8d9567.mov

Yes I did check it, here is a video I recorded for you to show the interaction. You can shift-click on any link in bangle to open it on the right side, which is similar to `Andy Matuschak's Notes`. Let me know what you think.

PS. Thanks for taking out time to open these issues <3

**Comment by RoyiAvital** [comment]
But what happens when you open another one on the right pane? It seems it will open on the left. Not like in the link.

**Comment by kepta** [comment]
Yes bangle does not  currently support more than two  editors, we might add it in future but not sure if it is actually helpful.
